src/components/Person/Person.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.scss';
+
+export const Person = (props) => {
+  const { name, age, sex, isMarried, partnerName } = props;
+
+  return (
+    <section className="person">
+      <h2 className="person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      {isMarried ? (
+        <p className="person__partner">
+          {sex === 'm'
+            ? `My wife's name is ${partnerName}`
+            : `My husband's name is ${partnerName}`
+          }
+        </p>
+      ) : (
+        <p className="person__partner">
+          I am not married
+        </p>
+      )}
```

Comment:
All you need is to set the text inside the `<p>` tag, so move your ternary inside it, or, better, move that logic to a variable